hooks/post-command

```diff
@@ -82,8 +82,12 @@ get_codecov_cli() {
 
   # One-time step
   if [[ "${BUILDKITE_PLUGIN_CODECOV_PGP_PUBLIC_KEY_URL-"false"}" = "" ]]; then
-    debug "Importing bundled Codecov's PGP public key : $(pwd)/pgp_keys.asc" 
-    cat "$(pwd)/pgp_keys.asc" \
+    local pgp_key_path=$(find "${BUILDKITE_PLUGINS_PATH}" -name pgp_keys.asc | grep codecov-buildkite-plugin | head -1)
```

Comment:
use the first result in case we multiple versions of the plugin exist on the agent

Reply:
```suggestion
    local pgp_key_path
    pgp_key_path=$(find "${BUILDKITE_PLUGINS_PATH}" -name pgp_keys.asc | grep codecov-buildkite-plugin | head -1)
```

Reply:
can we be sure the first hit is the correct one to compare with?

Reply:
well, we don't have multiple versions of the key so whichever one we pick should be good. we just need one of them.